application.php

```diff
@@ -17,6 +17,7 @@
         isset($_SERVER['jira_build_message']) ? $_SERVER['jira_build_message'] : null,
         isset($_SERVER['jira_url']) ? $_SERVER['jira_url'] : null,
         isset($_SERVER['jira_project']) ? $_SERVER['jira_project'] : null,
+        isset($_SERVER['jira_issue_key']) ? $_SERVER['jira_issue_key'] : null,
```

Comment:
The JIRA issue key would actually be an option. Here you pass it as an argument.
I think the proper way is would be `--jira_issue_key  $THE_KEY`